client/components/notice/notice-action.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, ReactNode } from 'react';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	'aria-label': string;
+	href: string;
+	onClick: () => void;
+	children: ReactNode;
+	external?: boolean;
+	icon?: string;
+}
+
+const NoticeAction: FunctionComponent< Props > = ( {
+	'aria-label': ariaLabel,
+	href,
+	onClick,
+	children,
+	external = false,
+	icon,
+} ) => {
+	const attributes = {
+		'aria-label': ariaLabel,
+		className: 'notice__action',
+		href: href,
```

Comment:
We could shorten this to:

```suggestion
		href,
```

Reply:
Good idea!

Reply:
Good idea!